packages/common/src/utils/wait.ts

```diff
@@ -1,3 +1,3 @@
 export function wait(ms: number): Promise<void> {
-  return new Promise((resolve) => setTimeout(resolve, ms));
+  return new Promise<void>((resolve) => setTimeout(() => resolve(), ms));
```

Comment:
not sure it's necessary, just slightly more explicit to avoid arguments passing through to resolve
(I had this in `waitForIdle` and felt more consistent to do it here too)